docs/references/ic-interface-spec.md

```diff
@@ -2121,7 +2121,9 @@ The optional `settings` parameter can be used to set the following settings:
 
 -   `freezing_threshold` (`nat`)
 
-    Must be a number between 0 and 2<sup>64</sup>-1, inclusively, and indicates a length of time in seconds.
+    Must be a number between 86400 (or equivalent of 1 day in seconds) and 2<sup>64</sup>-1, inclusively, and indicates a length of time in seconds.
```

Comment:
Why a hard lower bound instead of relying on the unsafe settings flag to allow very low freezing threshold? It could be useful to have a freezing threshold of zero, e.g., to move all cycles out of a canister.

Reply:
There's general agreement internally that a freezing threshold of 0 (or anything below a day) is extremely dangerous and the added value of saving a few more cycles (like in the use case you describe) does not outweigh the risks. In fact, some people wanted to completely disallow anything below 2 weeks (or even 30 days) but that certainly felt way too aggressive.

Reply:
Moving cycles out of a canister will anyway be facilitated by the new ic0.canister_liquid_cycle_balance128.